src/sentry/static/sentry/app/components/timeSince.jsx

```diff
@@ -18,35 +27,46 @@ var TimeSince = React.createClass({
     };
   },
 
+  getInitialState() {
+    return {
+      relative: this.getRelativeDate()
+    };
+  },
+
   componentDidMount() {
-    var delay = 2600;
+    this.setRelativeDateTicker();
+  },
 
-    this.ticker = setInterval(this.ensureValidity, delay);
+  setRelativeDateTicker() {
+    const ONE_MINUTE_IN_MS = 3600;
+
+    this.ticker = setTimeout(() => {
+      this.setState({
+        relative: this.getRelativeDate()
+      });
+      this.setRelativeDateTicker();
+    }, ONE_MINUTE_IN_MS);
```

Comment:
Yeah, that's true (it was doing that before, too).

We should make the timeout according to the duration of the relative time. Follow-up?